speedy-recovery-client/src/components/secured/patientinformation/PatientInfo.js

```diff
@@ -1,15 +1,311 @@
 import React, { Component } from "react";
+import { Icon, Label, Menu, Tab, Table } from "semantic-ui-react";
+import "./PatientInfo.css";
 
 class PatientInfo extends Component {
   render() {
-    const { user } = this.props;
+    const { user, conditions, medicationDispenses, carePlans } = this.props;
+    const conditionsNum = conditions.length;
+    const createConditionTable = () => {
+      let table = [];
+      if (conditionsNum) {
+        let header = [
+          <Table.Row key={"conditionrow"}>
+            <Table.HeaderCell id="patientTableCell">#</Table.HeaderCell>
+            <Table.HeaderCell id="patientTableCell">Summary</Table.HeaderCell>
+            <Table.HeaderCell id="patientTableCell" />
+            <Table.HeaderCell id="patientTableCell">Time</Table.HeaderCell>
+          </Table.Row>
+        ];
+        table.push(
+          <Table.Header id="patientTableHeader" key="conditionTableHeader">
+            {header}
+          </Table.Header>
+        );
+      }
+      let body = [];
+      for (let i = 0; i < conditionsNum; i++) {
+        let children = [];
+        let summary = conditions[i].summary;
+        let searchQuery = "https://www.google.com/search?q=" + summary;
```

Comment:
I think we might want to save this feature for the parents (please get in touch with the team implementing parents features and show them the code). In general, I think it might not be a good idea to let the young kids google their diseases.

Reply:
Lets discuss this in the meeting